content/en/docs/concepts/cluster-administration/networking.md

```diff
@@ -293,6 +293,78 @@ Weave Net runs as a [CNI plug-in](https://www.weave.works/docs/net/latest/cni-pl
 or stand-alone.  In either version, it doesn't require any configuration or extra code
 to run, and in both cases, the network provides one IP address per pod - as is standard for Kubernetes.
 
+## IPv4/IPv6 dual stack
+
+{{< feature-state state="alpha" >}}
+{{< warning >}}Kubernetes' implementation of dual-stack networking is in flux. You may well find that the design is changed in future releases.{{< /warning >}}
```

Comment:
The style guide [recommends](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-statements-about-the-future) avoiding statements that are tied to a point in time.
The alpha feature state should serve as enough warning, I hope.

Reply:
Perhaps `<feature-state state="alpha" for_k8s_version="v1.16">` would provide both a true-across-time statement, and further highlight the 1.16 requirement?